src/components/Navbar.js

```diff
@@ -1,101 +1,81 @@
 import github from "../img/github-icon.svg";
 import logo from "../img/logo.svg";
 
-const Navbar = class extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      active: false,
-      navBarActiveClass: "",
-    };
-  }
+const Navbar = () => {
+  const [isActive, setIsActive] = useState(false);
 
-  toggleHamburger() {
-    // toggle the active boolean in the state
-    this.setState(
-      {
-        active: !this.state.active,
-      },
-      // after state has been updated,
-      () => {
-        // set the class in state for the navbar accordingly
-        this.state.active
-          ? this.setState({
-              navBarActiveClass: "is-active",
-            })
-          : this.setState({
-              navBarActiveClass: "",
-            });
-      }
-    );
-  }
-
-  render() {
-    return (
-      <nav
-        className="navbar is-transparent"
-        role="navigation"
-        aria-label="main-navigation"
-      >
-        <div className="container">
-          <div className="navbar-brand">
-            <Link to="/" className="navbar-item" title="Logo">
-              <img src={logo} alt="Kaldi" style={{ width: "88px" }} />
-            </Link>
-            {/* Hamburger menu */}
-            <div
-              className={`navbar-burger burger ${this.state.navBarActiveClass}`}
-              data-target="navMenu"
-              role="menuitem"
-              tabIndex={0}
-              onKeyPress={() => this.toggleHamburger()}
-              onClick={() => this.toggleHamburger()}
-            >
-              <span />
-              <span />
-              <span />
-            </div>
-          </div>
-          <div
-            id="navMenu"
-            className={`navbar-menu ${this.state.navBarActiveClass}`}
+  return (
+    <nav
+      className="navbar is-transparent"
+      role="navigation"
+      aria-label="main-navigation"
+    >
+      <div className="container">
+        <div className="navbar-brand">
+          <Link to="/" className="navbar-item" title="Logo">
+            <img src={logo} alt="Kaldi" style={{ width: "88px" }} />
+          </Link>
+          {/* Hamburger menu */}
+          <button
+            className={`navbar-burger burger ${isActive && "is-active"}`}
+            aria-expanded={isActive}
+            tabIndex={0}
+            onKeyPress={() => setIsActive(!isActive)}
+            onClick={() => setIsActive(!isActive)}
           >
-            <div className="navbar-start has-text-centered">
+            <span />
+            <span />
+            <span />
+          </button>
+        </div>
+        <ul id="navMenu" className={` navbar-start has-text-centered navbar-menu ${isActive && "is-active"}`}>
+            {/* TODO: inline override of padding is a result of refactoring
+                to a ul for accessibilty purposes, would like to see a css
+                re-write that makes this unneccesary.
+             */}
```

Comment:
Ahhh this is makes a lot of sense that this change causes some friction. For this note, are you looking for suggestions? Would you want to pair on this? :smile:

Reply:
I just didn't want to start re writing the css in this PR, knowing that we intended to so some css restructuring in the future. I feared if I started, I wouldn't stop but I also wanted to make sure that there was a note explaining why this was done and that it should be addressed in the future.

Reply:
Works for me!